test-requirements.txt

```diff
@@ -1,2 +1,2 @@
-botocore>=1.25.0
-boto3>=1.22.0
+botocore>=1.29.0
```

Comment:
not mandatory, but this can be added as `trivial` in the changelog as we are bumping minimal required boto* versions